### PR TITLE
Update the README.md file

### PR DESCRIPTION
**Pull request by parauliya**
1. Update the links of boots, hegel 
2. Added osie, pbnj
3. Added Packet and Equinix reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://cloud.drone.io/api/badges/tinkerbell/tink/status.svg)](https://cloud.drone.io/tinkerbell/tink)
 
-Tinkerbell is a bare metal provisioning and workflow engine comprised of three major components: a DHCP server ([boots](https://github.com/packethost/boots)), a workflow engine (tinkerbell, this repository), and a metadata service ([hegel](https://github.com/packethost/hegel)). The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
+Tinkerbell is a bare metal provisioning and workflow engine. It was open-sourced by Packet, an Equinix company, and is comprised of four major components: a DHCP server ([boots](https://github.com/tinkerbell/boots)), a workflow engine (tink, this repository), a metadata service ([hegel](https://github.com/tinkerbell/hegel)) and an in-memory installation environment([osie](https://github.com/tinkerbell/osie)). The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
```
Comment (mmlb) [inline README.md:5]: can you split this to 2 commits please? One to split each sentence onto its own line and the second with your Equinix and other changes?

Comment (parauliya) [reply README.md:5]: sure

File: README.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
...
+It was open-sourced by Packet, an Equinix company.
+
+It is comprised of following four major components:
+
+ 1. A DHCP server ([boots](https://github.com/tinkerbell/boots))
+ 2. A workflow engine (tink, this repository)
+ 3. A metadata service ([hegel](https://github.com/tinkerbell/hegel))
+ 4. An in-memory installation environment([osie](https://github.com/tinkerbell/osie))
+
+ The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
```
Comment (mmlb) [inline README.md:15]: can you split this on `.` to be 2 lines.

Comment (grahamc) [inline README.md:15]:
```suggestion
 The workflow engine is comprised of a server and a CLI, which communicates over gRPC. The CLI is used to create a workflow and its building blocks: templates and targeted hardware.
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,17 @@
 
 [![Build Status](https://cloud.drone.io/api/badges/tinkerbell/tink/status.svg)](https://cloud.drone.io/tinkerbell/tink)
 
-Tinkerbell is a bare metal provisioning and workflow engine comprised of three major components: a DHCP server ([boots](https://github.com/packethost/boots)), a workflow engine (tinkerbell, this repository), and a metadata service ([hegel](https://github.com/packethost/hegel)). The workflow engine is comprised of a server and a CLI, which communicate over gRPC. The CLI is used to create a workflow along with its building blocks, templates and targeted hardware.
+Tinkerbell is a bare metal provisioning and workflow engine.   
+It was open-sourced by Packet, an Equinix company.
```
[the remark this answers is not in the document]
Comment (parauliya) [reply README.md:6]: seems good to change.

File: README.md (hunk 2 above)
Comment (parauliya) [reply README.md:15]: This one is also seems good to change.

Comment (parauliya) [reply README.md:15]: Done

File: README.md [hunk 4]
```diff
@@ -2,15 +2,27 @@
...
+ The CLI is used to create a workflow and its building blocks: templates and targeted hardware.
 
 # Packet Workflow 
 
 A Packet Workflow is an open-source microservice that’s responsible for handling flexible, bare metal
 provisioning workflows, that is...
  - standalone and does not need the Packet API to function
- - contains `Boots`, `Tinkerbell`, `Osie`, and workers
- - can bootstrap any remote worker using `Boots + Osie`
+ - contains `Boots`, `Tink`, `Osie`, `PBnJ` and workers
```
Comment (mmlb) [inline README.md:24]: Osie -> OSIE, missing Hegel.

File: README.md [hunk 5]
```diff
@@ -20,7 +32,8 @@ provisioning workflows, that is...
  - [Components](docs/components.md)
    - [Boots](docs/components.md#boots)
    - [Osie](docs/components.md#osie)
```
Comment (mmlb) [inline README.md:34]: Osie -> OSIE

File: README.md [hunk 6]
```diff
@@ -2,15 +2,27 @@
...
 
 # Packet Workflow 
 
 A Packet Workflow is an open-source microservice that’s responsible for handling flexible, bare metal
 provisioning workflows, that is...
  - standalone and does not need the Packet API to function
- - contains `Boots`, `Tinkerbell`, `Osie`, and workers
- - can bootstrap any remote worker using `Boots + Osie`
+ - contains `Boots`, `Tink`, `Osie`, `PBnJ` and workers
+ - can bootstrap any remote worker using `Boots + Osie + PBnJ`
```
Comment (mmlb) [inline README.md:25]: Osie -> OSIE

Comment (mmlb) [inline README.md:25]: Hegel too.

File: README.md (hunk 4 above)
Comment (parauliya) [reply README.md:24]: Done

File: README.md [hunk 7]
```diff
@@ -2,17 +2,24 @@
...
+ The CLI is used to create a workflow and its building blocks: templates and targeted hardware.
 
 # Packet Workflow
 
 A Packet Workflow is an open-source microservice that’s responsible for handling flexible, bare metal provisioning workflows, that is...
 
 -   standalone and does not need the Packet API to function
--   contains `Boots`, `Tinkerbell`, `Osie`, and workers
--   can bootstrap any remote worker using `Boots + Osie`
+-   contains contains `Boots`, `Tink`, `Hegel`, `OSIE`, `PBnJ` and workers
```
Comment (mmlb) [inline README.md:21]: `contains contains` dupe I assume?

**Pull request merged**